Plan.md

```diff
@@ -0,0 +1,3 @@
+-[] Set up Flask Server
```

Comment:
I like that your planning was shown but possibly a bit too vague if someone wanted to follow steps to repeat what you did